turbo.json

```diff
@@ -45,6 +45,10 @@
     "build:docs": {
       "outputs": ["README.md", "demo/index.*", "docs/partials/*.md"]
     },
+    "@auro-formkit/build-tools#build:docs:kit": {
+      "inputs": ["../../docs/partials/**/*", "../../docs/templates/**/*", "src/kitDocProcessor.mjs"],
```

Comment:
What directory does this path relatively direct from?

Reply:
These paths are relative to where the script is run from. In this case, from `packages/build-tools/`:

https://github.com/AlaskaAirlines/auro-form/blob/cfriedberg/11/packages/build-tools/package.json#L16

Reply:
The script is running from the root. Then should the paths to be `doc/...`?

Reply:
The script in the root uses Turbo which uses the `--filter` flag to target the package script:

https://github.com/AlaskaAirlines/auro-form/blob/cfriedberg/11/package.json#L35

This allows it to take advantage of Turbo's caching and features

Reply:
Confirmed `process.cwd()` on `npm run build:docs:kit` is `packages/build_tools` 😊